### PR TITLE
fix(#800): Set-IssueParent.ps1 attach-existing primitive + OrphanClaimWarnings orphan-drift detector

### PR DESCRIPTION
**Pull request by Grimblaz**
## Summary

Fixes the follow-up-issue orphan-drift bug: the documented "attach a follow-up to its umbrella" command was a silent no-op, so filed issues could land unattached with no signal anything went wrong. This PR implements both halves of the fix decided in design:

- **A2** — a new standalone `Set-IssueParent.ps1` script (attach-existing, extracted from `Add-FollowUpIssue.ps1`'s embedded GraphQL linking logic) that fails **loudly** (non-zero exit) instead of silently, and idempotently splices a detectable `Parent: #N` + `<!-- parent-link-mode: text-fallback -->` claim into the child body on failure so an unresolved attach is never invisible.
- **B1** — a warn-only `OrphanClaimWarnings` bucket in `render-portfolio.ps1`'s tracker renderer that flags any open issue claiming a parent but lacking the real sub-issue link, surfaced both in the tracker body and as a CI `::warning::` annotation — regardless of how the issue was originally filed.

Documentation drift fixed at all five sites in `skills/safe-operations/SKILL.md` that referenced the nonexistent `Add-FollowUpIssue` attach-existing capability, plus a `Documents/Design/control-tower-v2.md` update documenting the new warning tier.

This went through the full pipeline: `/experience` → `/design` (3-pass adversarial design challenge, 12 findings incorporated) → `/plan` (5-pass adversarial plan stress-test, 16 findings — one **critical**) → `/orchestrate` (implementation → 5-pass standard code review → fix cycle → CE Gate) → `/review-github` (proxy prosecution of external bot review).

**Two adversarial passes each caught a version of the same critical defect** — a naive regex anchor would have silently missed the exact real-world orphan format (`Board positioning: …; placement=parent #N; …`, written mid-line by §2b-ter) that motivated this issue in the first place. The design-phase stress-test caught it in the *design*; the plan-phase stress-test caught that even the corrected design's `(?m)` regex flag was itself wrong (matches every line, not just the first) when it reached the *implementation*. Both are fixed; see Adversarial Review below for the code-review-stage version of this same class of bug.

Closes #800

## Changed Files

- `skills/safe-operations/scripts/Set-IssueParent.ps1` (new) — attach-existing primitive
- `.github/scripts/Tests/Set-IssueParent.Tests.ps1` (new) — 9 tests
- `.github/scripts/Tests/set-issue-parent-doc-contract.Tests.ps1` (new) — 3 tests, AST-bound doc↔script contract
- `.github/scripts/render-portfolio.ps1` — `OrphanClaimWarnings` detector + body plumbing + CI `::warning::`
- `.github/scripts/Tests/render-portfolio.Tests.ps1` — +18 tests for the new detector
- `.github/scripts/Tests/safe-operations-2b-bis-contract.Tests.ps1` — tightened to bind the lever-table row specifically
- `skills/safe-operations/SKILL.md` — 5 doc drift sites fixed
- `Documents/Design/control-tower-v2.md` — new warning-tier documented
- `plugin.json`, `.claude-plugin/plugin.json`, `.claude-plugin/marketplace.json`, `.github/plugin/marketplace.json`, `README.md`, `CHANGELOG.md` — version bump 3.3.6 → 3.3.7 (release-gate, entry-point change)

## Validation Evidence

All test suites run and independently verified green at HEAD:

| Suite | Result |
|---|---|
| `Set-IssueParent.Tests.ps1` | 9/9 ✅ |
| `render-portfolio.Tests.ps1` | 92/92 ✅ |
| `set-issue-parent-doc-contract.Tests.ps1` | 3/3 ✅ |
| `safe-operations-2b-bis-contract.Tests.ps1` | 6/6 ✅ |
| `Add-FollowUpIssue.Tests.ps1` (regression — AC5, script untouched) | 5/5 ✅ |
| **Total** | **115/115 ✅** |

Both production scripts (`Set-IssueParent.ps1`, `render-portfolio.ps1`) parse cleanly (`[System.Management.Automation.Language.Parser]::ParseFile`, zero errors). `markdownlint-cli2 --fix` clean (0 errors) on both changed `.md` files. Newcomer-audit (warn-only): one advisory doc-anchor citation, non-blocking.

CI: `release-gate`, `pester`, `frame-enforce`, cost-pattern, and Sourcery all green.

## Review Mode

`standard` — full 5-pass two-layer prosecution panel (2 generalist + 3 specialist) → merged ledger → defense → judge, per `skills/adversarial-review/platforms/claude.md`. Plus a `/review-github` proxy-prosecution pass over the external bot review (see below).

## Adversarial Review Score Table (in-pipeline standard review)

16 findings prosecuted, 13 sustained, 3 defense-sustained (M8 perf-premise unevidenced, M10 proposed change would've been harmful, M14 factually-not-silent).

| Finding | Severity | Ruling | Disposition |
|---|---|---|---|
| M1 — pre-check hardcoded repo, writes used ambient `gh` context (cross-repo clobbering risk) | high | Sustained | ✅ Fixed |
| M2 — unconditional `Parent: #N` strip deleted `Add-FollowUpIssue`'s legitimate claim line | high | Sustained | ✅ Fixed |
| M3 — detector regex `(?m)` flag matched every line, not just the first (reintroduced the false-positive class the design eliminated) | medium | Sustained | ✅ Fixed |
| M4 — pre-check missing `sub_issues` GraphQL header + `$parsed.errors` check | medium | Sustained | ✅ Fixed |
| M5 — spurious edit from whitespace-only `TrimEnd()` diff | low | Sustained | ✅ Fixed |
| M6 — dual conflicting parent-claim numbers silently resolved to one | low | Sustained | ✅ Fixed |
| M7 — pre-check had no retry (asymmetric vs. the mutation's 2-attempt retry) | low | Sustained | ✅ Fixed |
| M8 — body-fetch "near-free" premise unevidenced | low | Defense-sustained | Dismissed (no failure mode) |
| M9 — stale "EXPECTED RED" test comments | low | Sustained | ✅ Fixed |
| M10 — regex short-circuit "optimization" | nit | Defense-sustained | Dismissed (would've degraded output) |
| M11 — no test asserted the warning renders in tracker markdown | low | Sustained | ✅ Fixed |
| M12 — untested `#unknown` interpolation branch | low | Sustained | ✅ Fixed |
| M13 — untested missing/closed-parent edge case | low | Sustained | ✅ Fixed |
| M14 — `--body` cmdline-length limit on huge bodies | low | Defense-sustained | Dismissed (failure is loud, not silent; unrealistic body size) |
| M15 — `2>$null` suppressed all `gh` stderr | nit | Sustained | ✅ Fixed |
| M16 — two-producer `parent-link-mode` marker-lifecycle divergence | low | Sustained | 📋 Deferred → [#827](https://github.com/Grimblaz/agent-orchestra/issues/827) |

**Judge confidence: high** — every ruling independently verified against the live tree, not taken from either ledger's say-so.

## GitHub Review Intake (`/review-github`)

External bot review (`gemini-code-assist`) raised 7 inline findings, all one root cause: the M15 fix used `2>$variable`, which does **not** capture stderr in PowerShell (it is a file-redirect operator, so with the variable `$null` at redirect time it discards stderr just like the original `2>$null`). Proxy prosecution → defense → judge sustained all 7 (deduped to one root-cause fix), downgraded severity high → medium (diagnostic-only; control flow and loud-failure behavior intact), and caught that the bot's own suggested fix (merge `2>&1` into one variable, then parse it as JSON) would be **unsafe** at 4 of the 7 sites — `gh` emits stderr notices even on successful exit-0 calls, so merging streams before parsing would corrupt JSON/GraphQL payloads and cause false failures. Fixed with `ErrorRecord`-type stream separation instead (commit `5c65604`). All 7 review threads answered with disposition replies.

## Prosecution Depth Summary

- **Design-challenge** (3-pass, prosecution-only): 12 findings sustained, all incorporated into the design.
- **Plan-stress-test** (5-pass standard, prosecution → defense → judge): 16 findings, 16 sustained (1 critical — a plan-authoring regex bug that would have defeated Class-B detection for the exact motivating incident format).
- **Code-review** (5-pass standard, prosecution → defense → judge): 16 findings, 13 sustained (12 fixed inline, 1 deferred to #827).
- **GitHub review intake** (proxy prosecution → defense → judge): 7 findings, all sustained (one root cause, fixed inline).

Total: 51 findings raised across four adversarial stages of this pipeline; 48 incorporated/fixed, 1 deferred with a tracked follow-up, 2 correctly dismissed by defense/judge scrutiny.

## CE Gate Result

Surface: CLI / script + tracker renderer (no GUI) — pre-set as `code-audit` by the design's own CE Gate readiness assessment; no live browser/interactive session applicable or available (`preview_list` confirmed empty).

| Scenario | Type | Verdict | Evidence type |
|---|---|---|---|
| S1 — documented attach path creates the parent link | Functional | ✅ PASS | code-audit |
| S2 — mis-invocation / attach-failure fails loudly, not silently | Functional | ✅ PASS | code-audit |
| S3 — tracker never silently lies about parentage ("LOUDLY flagged") | Intent | ✅ PASS | code-audit |
| S4 — doc↔script drift cannot recur unnoticed | Functional `[auto]` | ✅ PASS | code-audit |

All 4/4 PASS, independently re-run (not cited from prior reports) and cross-checked against the live source, including a genuine falsifiability check on S4 (confirmed the contract test would actually go RED on a doc regression).

## Pipeline Metrics

- Phases run this session: Experience-Owner, Solution-Designer, Issue-Planner, Code-Conductor (hub mode, full tier — scope classification announced `full` on direct evidence of all three completed upstream markers).
- Scope tier: full (announced, not asked — deterministic rubric).
- D9: fired, "Continue implementation" (auto-mode default, all upstream phases ran this session).
- Phase-containment: design-challenge 12/12 blocks emitted; plan-stress-test 16/16 blocks emitted; code-review 13/13 blocks emitted (all three surfaces verified `clean` via `phase-containment-emission-check.ps1`).

## Process Gaps Found

- Missed the per-step commit gate (`skills/step-commit/SKILL.md`) during initial dispatch of steps s1–s5 — recovered by committing the completed work as one consolidated commit (`46e985a`) rather than five granular step commits. No functional impact; reduces resume-point granularity if this session had been interrupted mid-implementation.
- The `<!-- plan-issue-800 -->` marker was initially omitted when the plan was first persisted (frontmatter posted without the literal marker comment) — caught and corrected via a same-session `PATCH` before `/orchestrate` began, so smart-resume never actually operated on the broken state, but worth noting as a near-miss in the plan-persistence step.
- The version bump was initially missed (release-gate failed on the `bump` leg) — resolved via `bump-version.ps1` (3.3.6 → 3.3.7) + CHANGELOG entry.

Closes #800

### CODE REVIEW

File: skills/safe-operations/scripts/Set-IssueParent.ps1 [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+try {
+    $repoNwo = gh repo view --json owner,name -q '.owner.login + "/" + .name' 2>$repoViewStderr
+} catch {
+    $repoNwo = $null
+}
+if ($LASTEXITCODE -ne 0 -or [string]::IsNullOrWhiteSpace($repoNwo)) {
+    $stderrText = if ($repoViewStderr) { (($repoViewStderr | Out-String).Trim()) } else { '' }
+    $suffix = if ($stderrText) { " stderr: $stderrText" } else { '' }
+    Write-Error "Could not resolve the current repository via 'gh repo view' (owner/name).$suffix"
+    exit 1
```
[the remark this answers is not in the document]
Comment (Grimblaz) [reply skills/safe-operations/scripts/Set-IssueParent.ps1]: **Disposition: INCORPORATE (sustained, medium severity — downgraded from high; diagnostic-only, no control-flow impact).** Fixed in 5c65604. Note: your suggested fix code (merge `2>&1` into one variable, then parse as JSON) would have been unsafe at this and 3 other sites — `gh` emits stderr notices even on successful exit-0 calls, and merging streams before parsing corrupts the JSON/GraphQL payload. Applied stream separation (filter by `ErrorRecord` type) instead. Full ruling: https://github.com/Grimblaz/agent-orchestra/pull/829#issuecomment-4927954498

File: skills/safe-operations/scripts/Set-IssueParent.ps1 [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+                if ($childIssueData) {
+                    $childId = $childIssueData.id
+                    $childBody = $childIssueData.body
+                    if ($childIssueData.parent) {
+                        $currentParentNumber = $childIssueData.parent.number
+                    }
+                    $precheckSuccess = $true
+                }
+            }
+        }
```
[the remark this answers is not in the document]
Comment (Grimblaz) [reply skills/safe-operations/scripts/Set-IssueParent.ps1]: **Disposition: INCORPORATE (sustained, medium severity — downgraded from high; diagnostic-only, no control-flow impact).** Fixed in 5c65604. Note: your suggested fix code (merge `2>&1` into one variable, then parse as JSON) would have been unsafe at this and 3 other sites — `gh` emits stderr notices even on successful exit-0 calls, and merging streams before parsing corrupts the JSON/GraphQL payload. Applied stream separation (filter by `ErrorRecord` type) instead. Full ruling: https://github.com/Grimblaz/agent-orchestra/pull/829#issuecomment-4927954498

File: skills/safe-operations/scripts/Set-IssueParent.ps1 [hunk 3]
```diff
@@ -0,0 +1,316 @@
...
+        $strippedBody = Remove-StaleParentClaim -Body $childBody
+        if ($strippedBody -ne $childBody) {
+            $editStderr = $null
+            gh issue edit $ChildIssueNumber --body $strippedBody 2>$editStderr | Out-Null
+            if ($LASTEXITCODE -ne 0) {
+                $stderrText = if ($editStderr) { (($editStderr | Out-String).Trim()) } else { '' }
+                $suffix = if ($stderrText) { " stderr: $stderrText" } else { '' }
+                Write-Warning "Failed to strip stale parent-link-mode marker from issue #$ChildIssueNumber body.$suffix"
+            }
+        }
```
[the remark this answers is not in the document]
Comment (Grimblaz) [reply skills/safe-operations/scripts/Set-IssueParent.ps1:225]: **Disposition: INCORPORATE (sustained, medium severity — downgraded from high; diagnostic-only, no control-flow impact).** Fixed in 5c65604. Note: your suggested fix code (merge `2>&1` into one variable, then parse as JSON) would have been unsafe at this and 3 other sites — `gh` emits stderr notices even on successful exit-0 calls, and merging streams before parsing corrupts the JSON/GraphQL payload. Applied stream separation (filter by `ErrorRecord` type) instead. Full ruling: https://github.com/Grimblaz/agent-orchestra/pull/829#issuecomment-4927954498

File: skills/safe-operations/scripts/Set-IssueParent.ps1 [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+# Ported from Add-FollowUpIssue.ps1:158-208 (GraphQL Node-ID resolution +
+# addSubIssue mutation + 2-attempt retry). Divergence: on failure this
+# script exits non-zero instead of only Write-Warning-ing.
+$parentId = $null
+$parentViewStderr = $null
+try {
+    $parentId = gh issue view $ParentIssueNumber --json id --jq .id 2>$parentViewStderr
+} catch {
+    $parentId = $null
+}
```
[the remark this answers is not in the document]
Comment (Grimblaz) [reply skills/safe-operations/scripts/Set-IssueParent.ps1:249]: **Disposition: INCORPORATE (sustained, medium severity — downgraded from high; diagnostic-only, no control-flow impact).** Fixed in 5c65604. Note: your suggested fix code (merge `2>&1` into one variable, then parse as JSON) would have been unsafe at this and 3 other sites — `gh` emits stderr notices even on successful exit-0 calls, and merging streams before parsing corrupts the JSON/GraphQL payload. Applied stream separation (filter by `ErrorRecord` type) instead. Full ruling: https://github.com/Grimblaz/agent-orchestra/pull/829#issuecomment-4927954498

File: skills/safe-operations/scripts/Set-IssueParent.ps1 [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+                if ($parsed -and $parsed.errors) {
+                    Write-Warning "GraphQL addSubIssue returned errors: $($parsed.errors | ConvertTo-Json -Compress)"
+                } else {
+                    $graphqlSuccess = $true
+                }
+            }
+        } catch {
+            # Try again
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Grimblaz) [reply skills/safe-operations/scripts/Set-IssueParent.ps1]: **Disposition: INCORPORATE (sustained, medium severity — downgraded from high; diagnostic-only, no control-flow impact).** Fixed in 5c65604. Note: your suggested fix code (merge `2>&1` into one variable, then parse as JSON) would have been unsafe at this and 3 other sites — `gh` emits stderr notices even on successful exit-0 calls, and merging streams before parsing corrupts the JSON/GraphQL payload. Applied stream separation (filter by `ErrorRecord` type) instead. Full ruling: https://github.com/Grimblaz/agent-orchestra/pull/829#issuecomment-4927954498

File: skills/safe-operations/scripts/Set-IssueParent.ps1 [hunk 6]
```diff
@@ -0,0 +1,316 @@
...
+    $strippedBody = Remove-StaleParentClaim -Body $childBody
+    if ($strippedBody -ne $childBody) {
+        $editStderr = $null
+        gh issue edit $ChildIssueNumber --body $strippedBody 2>$editStderr | Out-Null
+        if ($LASTEXITCODE -ne 0) {
+            $stderrText = if ($editStderr) { (($editStderr | Out-String).Trim()) } else { '' }
+            $suffix = if ($stderrText) { " stderr: $stderrText" } else { '' }
+            Write-Warning "Failed to strip stale parent-link-mode marker from issue #$ChildIssueNumber body.$suffix"
+        }
+    }
```
[the remark this answers is not in the document]
Comment (Grimblaz) [reply skills/safe-operations/scripts/Set-IssueParent.ps1:307]: **Disposition: INCORPORATE (sustained, medium severity — downgraded from high; diagnostic-only, no control-flow impact).** Fixed in 5c65604. Note: your suggested fix code (merge `2>&1` into one variable, then parse as JSON) would have been unsafe at this and 3 other sites — `gh` emits stderr notices even on successful exit-0 calls, and merging streams before parsing corrupts the JSON/GraphQL payload. Applied stream separation (filter by `ErrorRecord` type) instead. Full ruling: https://github.com/Grimblaz/agent-orchestra/pull/829#issuecomment-4927954498

File: skills/safe-operations/scripts/Set-IssueParent.ps1 [hunk 7]
```diff
@@ -0,0 +1,316 @@
...
+} else {
+    Write-Error "Failed to attach issue #$ChildIssueNumber to parent issue #$ParentIssueNumber."
+    $fallbackBody = Add-ParentClaimFallback -Body $childBody -ParentIssueNumber $ParentIssueNumber
+    $editStderr = $null
+    gh issue edit $ChildIssueNumber --body $fallbackBody 2>$editStderr | Out-Null
+    if ($LASTEXITCODE -ne 0) {
+        $stderrText = if ($editStderr) { (($editStderr | Out-String).Trim()) } else { '' }
+        $suffix = if ($stderrText) { " stderr: $stderrText" } else { '' }
+        Write-Warning "Failed to write parent-link-mode text-fallback claim to issue #$ChildIssueNumber body.$suffix"
+    }
```
[the remark this answers is not in the document]
Comment (Grimblaz) [reply skills/safe-operations/scripts/Set-IssueParent.ps1:317]: **Disposition: INCORPORATE (sustained, medium severity — downgraded from high; diagnostic-only, no control-flow impact).** Fixed in 5c65604. Note: your suggested fix code (merge `2>&1` into one variable, then parse as JSON) would have been unsafe at this and 3 other sites — `gh` emits stderr notices even on successful exit-0 calls, and merging streams before parsing corrupts the JSON/GraphQL payload. Applied stream separation (filter by `ErrorRecord` type) instead. Full ruling: https://github.com/Grimblaz/agent-orchestra/pull/829#issuecomment-4927954498
